src/platform_impl/macos/event.rs

```diff
@@ -132,18 +147,21 @@ pub(crate) fn create_key_event(
 
     let key_from_code = code_to_key(physical_key, scancode);
     let (logical_key, key_without_modifiers) = if matches!(key_from_code, Key::Unidentified(_)) {
+        // `key_without_modifiers` ignores all modifiers except for shift.
```

Comment:
it should ignore all including shift, and I think it does that.

Reply:
True, my bad!